utils/gh-update.py

```diff
@@ -97,15 +98,15 @@ def update_status(repo, commit, token, data):
     api_url = ("https://api.github.com/repos/%s/statuses/%s" %
                (repo, commit))
 
-    print "Updating status of commit", commit, "with data", data
+    print("Updating status of commit", commit, "with data", data)
```

Comment:
Using ``print()`` in Python 2 will actually work ..

```
$ python --version; python -c "print('hi')"
Python 2.7.12
hi
```

but it can't hurt to import from future. That will ensure no bare ``print`` statements get added later.